src/Mollie/WC/Payment/Order.php

```diff
@@ -1058,4 +1064,18 @@ public function refund_amount($order, $amount, $payment_object, $reason)
 
 		return false;
 	}
+
+    /**
+     * @param Mollie\Api\Resources\Order $order
+     * @param string                     $order_id
+     */
+    protected function updatePaymentAfterPaid($order, $order_id)
```

Comment:
Not quite sure to understand the method, it does payment and then update it or it is called this way because it's called somewhere after the payment has been done? If so why?

Reply:
Yes, this name is not good as it leads to confusion. This method updates the payment with the necessary data to make the refund call from mollie dashboard possible when there is a refund or a chargeback in the payment (the core of this issue) so we need to do it after its paid. updatePaymentForCallbacks? updatePaymentForDashboardActions?

Reply:
I saw you changed the name to `updatePaymentForDashboardActions` which is quite confusing because `dashboard` and `actions` in this context are not explained, we don't know which dashboard and which actions. Also this lead to more questions about why the payment is connected to a dashboard and if this save info for a dashboard only and if the actions are related to the dashboard or are other things. 

The name should reflect what is inside the method not what you may thing it does in a more big context.